pkg/resource/aws/testdata/acc/aws_lambda_event_source_mapping/main.tf

```diff
@@ -88,11 +88,11 @@ resource "aws_iam_policy_attachment" "policy_attachment" {
 }
 
 resource "aws_lambda_function" "test_lambda" {
-  filename         = "function.zip"
-  function_name    = "event-source-mapping-test-lambda-${local.timestamp}"
-  role             = aws_iam_role.iam_for_lambda.arn
-  handler          = "exports.test"
-  runtime       = "nodejs12.x"
+  filename      = "function.zip"
+  function_name = "event-source-mapping-test-lambda-${local.timestamp}"
+  role          = aws_iam_role.iam_for_lambda.arn
+  handler       = "exports.test"
+  runtime       = "nodejs14.x"
```

Comment:
what's the reason this one is node 14 and line 89 at pkg/resource/aws/testdata/acc/aws_apigatewayv2_authorizer/terraform.tf is node 18? can't spot the difference and the original one are both the same version